### PR TITLE
Remove redundant function and modify comments in the PW module

### PR DESCRIPTION
**Pull request by yang2022i**
1.We remove redunant function.

The class “big” is temporary and ensures that the dimensions of the FFT can be evenly divided by bx, by, and bz. This is a requirement due to the current lattice integrals. In reality, lattice integrals can be done without this requirement, and in that case, this temporary class can be directly removed.

_________________________________________________________________________
2.We modify the comment to doxygen format

### CODE REVIEW

File: a [hunk 1]
```diff
@@ -0,0 +1,49 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
Comment (mohanchen) [inline a]: this file does not belong to this PR?

File: a.pub [hunk 2]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC/V7ksxnCxzzq1oL51n0Gssk21wjNFVoNH99VzmWXs/16HQpBgAUrrnUXOG+a9aAe9hBQtFpLvh7ylRCJ767nIsyiA+2WEUSreHEaLhc2szoUjnLDMpfgLUVs79mXG0L58qmks1NSY3MuHhmnKUkGvOPH64AreW4Ky3EwMNS4/fc5WMyq5sIoOsP+LlFY9jJjkGjHbPgygb/KDiU+OrsPVSX/ipqxvvxCKIGFTgoOSyCb8Q3AV/28qiOjZwGBgYUZRJ49K6Fjma6x23raa+d22128idyMXVMfl67sZxuMIDzmndKsCJCJ1WHsT7oEBFsoFefYWHH7totT/mLT4mCJTpnpSUcyh+/mSHlxEoWCwZr7P847kk7XzVXr0HIUDdHjeIAvO+dc5UZVMEAf4HP1bsIEyWmUxPvNMq+2ZTH1ZnEH6CLDT3x4kUNW/aZPJARejB0SlRNPgjzSDBOkr68vbJwVci5vUp0aAY7RdcOFH6dMv3ipLZVKzDQb0Ph3vvqUwXfa2w8/k8679w2GNyOEgPtd39jBndE5JoejsfK2a1wQAm698PGRKQkojPruLltEJrcPbwB9MuQc1Na/dbJPwvOwZQ3dZ9CAzSMrkq6Rc8h1YSgid8urX5rJ9M+ZQAjLISnsDkM1ksPSDH7xWjlzgX9izMJf5Opsp+knxwFTrGw== guoyang@雁的RedmiBook
```
Comment (mohanchen) [inline a.pub]: this file does not belong to this PR?

File: a (hunk 1 above)
Comment (yang2022i) [reply a]: Fixed. Thx!

File: a.pub (hunk 2 above)
Comment (yang2022i) [reply a.pub]: Fixed. Thx!

**Review by mohanchen — APPROVED** [review]
